src/components/FileList/CopyrightsRenderer/CopyrightsRenderer.css

```diff
@@ -0,0 +1,3 @@
+.copyrightPopover{
```

Comment:
I preferred to keep the style attached to the component so it could be easily found and managed. Not a big deal to move on another place for consistency